github-config.yaml

```diff
@@ -595,7 +595,6 @@ orgs:
           learn-katacoda: admin
           log-anomaly-detector: admin
           ludus: admin
-          manage-dependencies-tutorial: admin
```

Comment:
@pacospace please dont remove them from the SourceOps team, as the bots still need to be maintainers, as the assist us humans.